packages/cli/test/integration/generators/example.integration.js

```diff
@@ -49,7 +49,7 @@ describe('lb4 example', function() {
       .executeGenerator(generator)
       .withPrompts({name: VALID_EXAMPLE})
       .then(() => {
-        const targetPkgFile = `loopback4-example-${VALID_EXAMPLE}/package.json`;
+        const targetPkgFile = 'package.json';
```

Comment:
❗️ yeoman test generator / runner context is now moved to `loopback4-example-${VALID_EXAMPLE}` and hence we are now only looking for `package.json` (we are in the folder).